### ISSUE TITLE
cr.tl.lineages ValueError

### ISSUE THREAD

**Issue by rekharichardson1** [theislab/cellrank#1091]
hi! I am having an error trying to run cr.tl.lineages. Is this a dependency problem? Previously, I was able to run this code fine, and now I am not able to. 

```python
...
```

```pytb

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[170], line 1
----> 1 cr.tl.lineages(adata, solver='direct')
      2 cr.pl.lineages(adata, same_plot=False)

File [/opt/homebrew/lib/python3.9/site-packages/cellrank/tl/_utils.py:1643](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/lib/python3.9/site-packages/cellrank/tl/_utils.py:1643), in _deprecate..wrapper(wrapped, instance, args, kwargs)
   1636     warnings.simplefilter("always", DeprecationWarning)
   1637     warnings.warn(
   1638         f"`cellrank.tl.{wrapped.__name__}` will be removed in version `{version}`. "
   1639         f"Please use the `cellrank.kernels` or `cellrank.estimators` interface instead.",
   1640         stacklevel=2,
   1641         category=DeprecationWarning,
   1642     )
-> 1643 return wrapped(*args, **kwargs)

File [/opt/homebrew/lib/python3.9/site-packages/cellrank/tl/_lineages.py:71](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/lib/python3.9/site-packages/cellrank/tl/_lineages.py:71), in lineages(adata, backward, copy, return_estimator, **kwargs)
     66     raise RuntimeError(
     67         f"Compute the states first as `cellrank.tl.{fs_key}(..., backward={backward})`."
     68     )
     70 # compute the absorption probabilities
---> 71 mc.compute_absorption_probabilities(**kwargs)
     73 return mc.adata if copy else mc if return_estimator else None

File [/opt/homebrew/lib/python3.9/site-packages/cellrank/tl/estimators/mixins/_absorption_probabilities.py:288](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/lib/python3.9/site-packages/cellrank/tl/estimators/mixins/_absorption_probabilities.py:288), in AbsProbsMixin.compute_absorption_probabilities(self, keys, solver, use_petsc, time_to_absorption, n_jobs, backend, show_progress_bar, tol, preconditioner)
...
    423     )
    424 mask = np.isclose(abs_classes.sum(1), 1.0, rtol=1e-3)
    425 if not np.all(mask):

ValueError: `10` value(s) are negative. Try decreasing the tolerance as `tol=...`, specifying a preconditioner as `preconditioner=...` or use a direct solver as `solver='direct'` if the matrix is small.

```

#### Versions:

cellrank==1.5.1 scanpy==1.9.1 anndata==0.8.0 numpy==1.22.4 numba==0.55.2 scipy==1.8.1 pandas==1.4.3 pygpcca==1.0.3 scikit-learn==1.1.1 statsmodels==0.13.2 python-igraph==0.9.11 scvelo==0.2.4 pygam==0.8.0 matplotlib==3.5.2 seaborn==0.11.2
>

**Comment by Marius1311** [comment]
Hi @rekharichardson1, you can try installing CellRank from main, the issue should be solved there. See the installation instructions under "Development version": https://cellrank.readthedocs.io/en/latest/installation.html

However, note that this will change a lot of things, for example, all cellrank.tl functions have been decrecated. See here for on overview of the changes we're making in CellRank version 2: https://cellrank.readthedocs.io/en/latest/about/version2.html

**Comment by michalk8** [comment]
Should be solved in 2.0, closing this.
